cmd/release-controller/bugzilla.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/golang/glog"
+	v1 "github.com/openshift/api/image/v1"
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+)
+
+// syncBugzilla checks whether fixed bugs in a release had their
+// PR reviewed and approved by the QA contact for the bug
+func (c *Controller) syncBugzilla(key queueKey) error {
+	defer func() {
+		err := recover()
+		panic(err)
+	}()
+
+	release, err := c.loadReleaseForSync(key.namespace, key.name)
+	if err != nil || release == nil {
+		return err
+	}
+
+	// check if verifyBugs publish step is enabled for release
+	var verifyBugs *PublishVerifyBugs
+	for _, publishType := range release.Config.Publish {
+		if publishType.Disabled {
+			continue
+		}
+		switch {
+		case publishType.VerifyBugs != nil:
+			verifyBugs = publishType.VerifyBugs
+			break
```

Comment:
not a big deal but break won't stop the loop.  doesn't matter for now, but could be an issue in the future (two verify bug jobs).